src/App.jsx

```diff
@@ -284,7 +287,25 @@ const MainContent = ({
   page,
   setPage,
   scrollTarget,
+  totalElements,
 }) => {
+  /**
+   * function to calculate and show correct results for relevant page.
+   *
+   * @param {number} resultsLength - the amount of results per page.
+   *
+   * @return {string} - returns a string with the amount of results
+   */
+  const amountOfResults = resultsLength => {
```

Comment:
You could be a smarty pants here and return an object as start and end values and piece them together in the template, it would make the utility function more useful as its completely decoupled from the template. 

eg:

```js
export default function App() {
  const amountOfResults = () => {
    return [10, 20];
  };

  const [ start, end ] = amountOfResults();

  const results = amountOfResults();
 
  return (
    <div className="App">
      <p>
        {start} - {end}
      </p>
      <p>{results.join(" - ")}</p>
    </div>
  );
}

```